### PR TITLE
Donate page with form

### PR DESCRIPTION
**Pull request by denys-medynskyi**
### Description

This page should contain a page where person should agree to public agreement before he donates money.

### How to test instructions

Visit http://localhost:3000/donate

### CODE REVIEW

File: app/assets/javascripts/app/donate_form.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    // form submit is preventing user from submitting form.
+    var toggleBtns = function() {
+      var paymentBtn = document.getElementsByClassName('js-donate-form-payment-btn')[0];
+      var validationBtn = document.getElementsByClassName('js-donate-form-validation-btn')[0];
+      toggleElement(paymentBtn);
+      toggleElement(validationBtn);
+    }
+
+    if(agreeToTermsCheckbox) {
+      agreeToTermsCheckbox.addEventListener( 'change', toggleBtns);
```
Comment (floor114) [inline app/assets/javascripts/app/donate_form.js]: Remove space before `'change'`

File: app/views/donate/show.html.slim [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<script type="text/javascript" id="widget-wfp-script" src="https://secure.wayforpay.com/server/pay-widget.js?ref=button"></script>
+<script type="text/javascript">function runWfpWdgt() {var wayforpay = new Wayforpay(); wayforpay.invoice('https://secure.wayforpay.com/button/b644556e003e4');}</script>
```
Comment (floor114) [inline app/views/donate/show.html.slim]: I'd move click logic from `onclick` parameter to `donate_form.js`, and remove that `_btn.slim` partial.

**Review by floor114 — DISMISSED** [review]

File: app/views/donate/show.html.slim (hunk 2 above)
Comment (denys-medynskyi) [reply app/views/donate/show.html.slim]: Will try do cleanup

**Comment by denys-medynskyi** [comment]
@floor114 thanks for feedback 🙇

**Pull request merged**